src/openclaw_pipeline/unified_pipeline_enhanced.py

```
@@ -1284,6 +1464,112 @@ def _load_latest_qualified_files(self) -> list[str]:
+                directory=directory,
+                recent=recent,
+                dry_run=dry_run,
+                auto_promote=True,
+                promote_threshold=1,
+                progress_callback=self._build_absorb_progress_callback(
+                    total_files=total_files,
+                    completed_before=completed_before,
+                    failed_before=failed_before,
+                ),
+            )
+        except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)